hawc/apps/mgmt/migrations/0003_migrate_data.py

```diff
@@ -0,0 +1,87 @@
+# Generated by Django 5.0.6 on 2024-08-16 19:29
+
+from django.db import migrations
+
+# hardcoded due to possible changes/removal of constants
+statuses = [(10, "Not Started"), (20, "Started"), (30, "Completed"), (40, "Abandoned")]
+
+types = [(10, "Preparation"), (20, "Data Extraction"), (30, "QA/QC"), (40, "Study Evaluation")]
+
+
+def migrate_tasks(apps, schema_editor):
+    Task = apps.get_model("mgmt", "Task")
+    TaskType = apps.get_model("mgmt", "TaskType")
+    TaskStatus = apps.get_model("mgmt", "TaskStatus")
+    Assessment = apps.get_model("assessment", "Assessment")
+
+    # create all possible types/statuses for each assessment
+    status_instances = []
+    type_instances = []
```

Comment:
My understanding is that we want users to specify tasks/statuses on assessment creation, so I created entries for all possible items per assessment. Creating only the statuses currently used in the Tasks table would lead to handling missing statuses when assigning triggers in the next migration